.github/workflows/docs.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           git config user.email "${GITHUB_ACTOR}@bots.github.com"
           git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*
       - name: Set up Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v2.2.2
```

Comment:
Is this desired? To my understanding, the old version would have installed the latest v2.* anyway.

Reply:
correct, they are the same.